### ISSUE TITLE
Error messages during zsh shell installation.

### ISSUE THREAD

**Issue by Vanals** [makersacademy/problem-solving#119]
Following the thread(http://jilles.me/badassify-your-terminal-and-shell/) for the installation of zsh I executed in my terminal  **$ brew install zsh**. During the installation, I got two errors. I tried to google them but without success.
I'm going to link the text that is left in the terminal after I ran the installation string.

LINK = https://nopaste.xyz/?a6dbb6a627fa3cc1#SIZgd/3NVK3JAsE/KkQ5pvQ9yX/exwjs/1nTbdiDHUc=

So as you can see at the end I got this two errors
Coloured screenshot =  https://ibb.co/f7Bnok
By the way, after this two messages, it says " You can try again using .." And then the installation seems goes fine.
This is the last part of the text, after the two error messages:
> You can try again using:
  brew link pcre
==> Summary
🍺  /usr/local/Cellar/pcre/8.41: 204 files, 5.3MB
==> Installing zsh
==> Downloading https://homebrew.bintray.com/bottles/zsh-5.4.2_1.sierra.bottle.t
==> Downloading from https://akamai.bintray.com/e1/e12f51411a259c9392384f4bc552d
######################################################################## 100.0%
==> Pouring zsh-5.4.2_1.sierra.bottle.tar.gz
🍺  /usr/local/Cellar/zsh/5.4.2_1: 1,392 files, 11.9MB

I'm not confident with the terminal so I want just be sure that all the installation is gone fine and that it is not necessary fix anything. 
Anyway, I continued the installation and everything seems ok.

Thank you

**Comment by majdeddine** [comment]
Hey @Vanals  I think the issue is with the homebrew folder permission. I suggest running $ brew doctor 
In your terminal to check the warnings.

**Comment by Vanals** [comment]
Thanks @majdeddine. Anyway, do you think that the installation of zsh has been done properly? Then I did as you said and this is the output https://ibb.co/hzKbWG .
Shall I follow the instruction? The terminal doesn't say nothing about permission.

**Comment by majdeddine** [comment]
First go to /usr/local/ and check the permission for share and bin .

**Comment by majdeddine** [comment]
Then you can follow what the doctor sad :
Brew link pcre (same thing for gdbm)
Or you can go a step further and follow this solution here : https://stackoverflow.com/questions/29319378/cant-link-pcre-thru-brew-in-max-os-yosemite

**Comment by Vanals** [comment]
I'm quite new so I haven't really found the folder local and bin. How can I find it?
By the way, I ran "sudo chown -R `whoami` /usr/local" into the terminal as the link said. But no output came out, and then I executed the links with gdbm and pcre. 
Now typing 'brew doctor' it says that "Your system is ready to brew.". Should I then have to run a particular command to let's install what before has been missed? Thanks ;)
